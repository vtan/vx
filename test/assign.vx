(
  (let a (+ (- 12 4) 8))
  (let b (+ 0xbe 0XEF00))
  (let c (+ (+ 0x41 0x4200) (+ 0x43_0000 0x4400_0000)))
  (let d 0x1000)
  (set d (+ d (- a 1)))
)
