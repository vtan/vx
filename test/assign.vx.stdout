00000000: 0f10 0000 0000 0000 4142 4344 0000 0000  ........ABCD....
00000010: beef 0000 0000 0000 1000 0000 0000 0000  ................
